### ISSUE TITLE
Limit of message

### ISSUE THREAD

**Issue by Mifrill** [Mifrill/chat#36]
200 symbols for example

**Comment by kir486680** [comment]
I am ready to fix it !

**Comment by Mifrill** [comment]
@kir486680 ok, feel free to make it, Thank you

**Comment by MikaelAbehsera** [comment]
@Mifrill  is this still open !

**Comment by Mifrill** [comment]
@MikaelAbehsera yeah, thanks. It seems @kir486680 was planning to do this, isn't it?

**Comment by MikaelAbehsera** [comment]
It does seem like that, but its been couple of months, so Im not sure if he still working on it @kir486680
